commerce_coordinator/apps/commercetools/catalog_info/edx_utils.py

```diff
@@ -56,9 +56,11 @@ def get_edx_successful_payment_info(order: CTOrder):
     return None, None
 
 
-# TODO update function and its return value name
-# the return value could be either stripe payment intent id or PayPal order ID
-def get_edx_payment_intent_id(order: CTOrder) -> Union[str, None]:
+def get_edx_psp_payment_id(order: CTOrder) -> Union[str, None]:
+    """
+    Retrieve the payment service provider (PSP) payment ID from a given
+    order which could be either a Stripe payment intent ID or a PayPal order ID.
```

Comment:
Can there be more payment service providers in the future? If yes, we can change this to a map format like this
```suggestion
    Retrieve the payment service provider (PSP) payment ID from an order.

    Currently supports:
        - Stripe: Payment Intent ID ("stripe_payment_intent_id")
        - PayPal: Order ID ("paypal_order_id")
```